src/TesterInternal/TesterInternal.csproj

```diff
@@ -132,6 +141,13 @@ set Configuration=$(Configuration)
 call "$(SolutionDir)Build\PostBuild.cmd"
 </PostBuildEvent>
   </PropertyGroup>
+  <Import Project="$(SolutionDir)\.nuget\NuGet.targets" Condition="Exists('$(SolutionDir)\.nuget\NuGet.targets')" />
```

Comment:
I think that's the auto-restore thing that VS keeps injecting now after @jthelin removed it. I think it's no harm.